lib/parse.rb

```diff
@@ -21,4 +24,18 @@ def get_line
       return @lines[0]
    end
 
+   def count_lines
+      @line_count = @lines.size
+      convert_to_json()
+   end
+
+   def convert_to_json
+      my_object = {
+         "#{@path}": {
+         lines: @line_count
```

Comment:
@wesjrock essa função `convert_to_json` só irá funcionar caso vocë chame a função `count_lines` antes, certo?
Igual você fez no seu teste:

```ruby
parse = Parse.new("../../log/games.log")
parse.fopen
my_json_object = parse.count_lines
 ```

Eu não vejo necessidade atribuir esse valor a variável de instância `@line_count`. Minha sugestão é de você tornar essa função `count_lines` privada:

```suggestion
private
def count_lines
  @lines.size
end
```

E a chamar dentro da função `convert_to_json`

```suggestion
         lines: count_lines()
```